addons/base_gengo/wizard/base_gengo_translations.py

```diff
@@ -60,7 +60,9 @@ def gengo_authentication(self, cr, uid, context=None):
         '''
         user = self.pool.get('res.users').browse(cr, 1, uid, context=context)
         if not user.company_id.gengo_public_key or not user.company_id.gengo_private_key:
-            return (False, _("Gengo `Public Key` or `Private Key` are missing. Enter your Gengo authentication parameters under `Settings > Companies > Gengo Parameters`."))
+            action_id = self.pool['ir.model.data'].xmlid_to_res_id(cr, uid, 'base.action_res_company_form')
```

Comment:
I think no need to pass context to this method as  xmlid_to_res_id method don't have context as an argument